replicator/Cargo.toml

```diff
@@ -12,5 +12,10 @@ telemetry = { workspace = true }
 anyhow = { workspace = true, features = ["std"] }
 thiserror = { workspace = true }
 rustls-pemfile = { workspace = true, features = ["std"] }
+sqlx = { workspace = true, features = [
+    "runtime-tokio-rustls",
+    "postgres",
+    "migrate",
+] }
```

Comment:
Needed to run migrations on the source database to create metadata objects in db.